_posts/2023-07-04-foss4g-2023.md

```diff
@@ -0,0 +1,45 @@
+---
+author: Jody Garnett
+layout: post
+title: GeoServer at FOSS4G 2023
+date: 2023-07-04
+categories:
+- Announcements
+tags:
+- conferences
+- foss4g
+- presentations
+- workshop
+---
+
+The GeoServer team was pleased to attend [FOSS4G 2023](https://2023.foss4g.org/) last week (26 June - 2 July 2023) in Prizren, Kosovo!
+
+FOSS4G 2023 was put on this year by the Open Source Geospatial Foundation ([OSGeo](https://www.osgeo.org/)) and the regional Free Libre Open Source Software Kosova ([FLOSSK](https://flossk.org)) organization. The [local organizing committee](https://2023.foss4g.org/about/team/) worked very hard to make everyone feel welcome and well cared for. Thank you for hosting everyone, we had a great time.
+
+GeoServer content:
+
+* [State of GeoServer 2.23](https://docs.google.com/presentation/d/1nRKIILXWGLMGXZ6thfJgPR9kZ6Wh8Hp1dwZdQGw2YRc/edit?usp=share_link)
+* [GeoServer Feature Frenzy 2023](https://docs.google.com/presentation/d/1vE8eCrOyewoH54g8CjuoiO3pxVLToEpuvpoZWmy0wTg/edit?usp=share_link)
+* [GeoServer used in fun and interesting ways](https://docs.google.com/presentation/d/1PP2qk7eH8TzAf1tvEWH7Geattd0YFh7ZEDx1_tlrRWY/edit?usp=share_link)
+* [GeoServer Orientation](https://docs.google.com/presentation/d/1sh9C4dIkDRnk3quCD1PRYoiJhjI9dqnAdOScJCgQWU8/edit?usp=share_link)
+* [Getting Started with GeoServer](https://docs.google.com/presentation/d/1FL6uMsepPWrrCMWjelIrlIp7bnlxB2q294-TN4lgI34/edit?usp=share_link) (Workshop)
+
+GeoServer community content:
+
+* Many additional presentations featured GeoServer and showcased its capabilities.
+
+If videos are provided of the event we will update this post with links.
+
+![](/img/posts/foss4g/prizren.jpeg) <br/>
+Prizren (Kosovo)
+
+## FOSS4G Code sprint
+
+We also had our first scheduled codesprint on removing the opengis-pending content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget. 
```

Comment:
Not sure "opengis-pending" is well enough known to explain it. 

```suggestion
We also had our first scheduled codesprint on r[efactoring the OpenGIS](https://www.osgeo.org/opengis-harmonization/) content from our codebase. The foss4g sprint was a fact-finding mission to establish the scope of work and establish a time frame and budget. 
```

Reply:
`r[efactoring` ?

Reply:
I've fixed the link, but I'm not in love with the expression.